nerium/contrib/resultset/jinja.py

```diff
@@ -0,0 +1,39 @@
+import os
+
+import records
+from dotenv import load_dotenv
+from nerium.contrib.resultset.sql import SQLResultSet
+
+try:
+    from jinjasql import JinjaSql
+except ImportError:
+    # jinjasql >= 0.1.7 required for use with repeat param def in template
+    pass
+
+
+class JinjaSQLResultSet(SQLResultSet):
+    """ Use jinjasql to process template for use against records db connection.
+    Similar to a regular SQLResultSet, but preprocesses the query with jinjasql
+    to allow for more complex conditional logic.
+
+    NOTE: jinjasql>=0.1.7 required for module usage.
+    """
+
+    def result(self):
+        try:
+            backend_path = self.query_path.parent
+            backend = self.backend_lookup(backend_path)
+            db = records.Database(backend)
+            try:
+                jinja = JinjaSql(param_style='named')
+            except NameError:
+                raise Exception("jinjasql >= 0.1.7 required for use")
```

Comment:
Do we really want to `raise` here? I would think we'd want to set `result` to the error message, as we do with other exceptions.

Reply:
By raising here we get that behavior actually:
`[
    {
        "error": "Exception('jinjasql >= 0.1.7 required for use',)"
    }
]`

rather than no try except which returns:
`[
    {
        "error": "NameError(\"name 'JinjaSql' is not defined\",)"
    }
]`

Reply:
Both will have response code 400

Reply:
👍 Great!